.travis.yml

```diff
@@ -227,7 +227,7 @@ install:
           export SYMFONY_VERSION=$(git ls-remote --heads | grep -o '/[1-9].*' | tail -n 1 | sed s/.//) &&
           git fetch --depth=2 origin $SYMFONY_VERSION &&
           git checkout -m FETCH_HEAD &&
-          export COMPONENTS=$(find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -printf '%h\n' | sort)
+          export COMPONENTS=$(find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -not -wholename '*/Component/Debug/*' -printf '%h\n' | sort)
```

Comment:
I think we can remove this - the failure that is triggered in relation here might be fixed by merging. If not, we'll apply.